samples/cors/ts/package.json

```diff
@@ -10,6 +10,6 @@
   "devDependencies": {
     "concurrently": "^5.3.0",
     "typescript": "^4.0.3",
-    "prettier": "^1.16.4"
+    "prettier": "^2.5.1"
```

Comment:
@maorleger Do you recall why we added prettier as a dependency here if we dont have any scripts using it?

Reply:
I have a hunch I added it so that my VSCode Prettier plugin uses the same version our SDK was using at the time... I don't think it's needed though. I probably just didn't know any better

Reply:
@maorleger would you recommend removing the dependency from the samples? I think that if we leave it on the samples, then new customers will know that our code formats to prettier(?), I'm not sure if that use case is realistic to be honest.

Reply:
I would recommend removing it

Reply:
Sorry, yes - let's remove it

Reply:
Removed :)